package.json

```diff
@@ -24,4 +24,4 @@
     "url": "https://github.com/IonicaBizau/node-w-json/issues"
   },
   "homepage": "https://github.com/IonicaBizau/node-w-json"
-}
+}
```

Comment:
That's written by [`blah`](https://github.com/IonicaBizau/node-blah). And `blah` uses `w-json`. You got it. :stuck_out_tongue_winking_eye: 

[Here](https://github.com/IonicaBizau/node-blah/blob/master/lib/index.js#L438) is should be:

```js
WriteJson(self.paths.pack, self.pack, true, next);
```

Not a big thing, anyway. :smile: